app/services/vacancy_alert_filters.rb

```diff
@@ -0,0 +1,17 @@
+class VacancyAlertFilters < VacancyFilters
+  AVAILABLE_FILTERS = %i[keyword].concat(superclass::AVAILABLE_FILTERS).freeze
+
+  attr_reader(*AVAILABLE_FILTERS)
+
+  def initialize(args)
+    args = ActiveSupport::HashWithIndifferentAccess.new(args)
+
+    @keyword = args[:keyword]
+
+    super(args)
+  end
+
+  def to_hash
+    super().merge(keyword: keyword)
```

Comment:
I don't think you need the `()` if there's no args to pass to `super`

Reply:
My understanding of what I read was that `super.merge()` would try to call the `merge` method on the superclass. Did I get that wrong?

Reply:
Nah, that's right, I just don't think you need the brackets as there's no arguments to the method in the superclass. I could be wrong like (I frequently am 😆 )